src/main/java/Duke.java

```diff
@@ -1,10 +1,139 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        Task[] tasks = new Task[100];
+        boolean isFinished = false;
+        String horizLine = "____________________________________________________________";
+        showWelcomeMessage(horizLine);
+        executeUserCommand(tasks, horizLine);
+
+    }
+
+    private static void executeUserCommand(Task[] tasks, String horizLine) {
+
+        String command;
+        Scanner in = new Scanner(System.in);
+        command = in.nextLine();
+        int taskCount = 0;
+
+        while (!command.equals("bye")) {
+            if (command.contains("todo")) {
+                taskCount = computeTodoCommand(tasks, horizLine, command, taskCount);
+            }
+
+            else if (command.contains("deadline")) {
+                taskCount = computeDeadlineCommand(tasks, horizLine, command, taskCount);
+            }
+
+            else if (command.contains("event")){
+                taskCount = computeEventCommand(tasks, horizLine, command, taskCount);
+            }
+
+            else if (command.equals("list")) {
+                printList(tasks, horizLine, taskCount);
+            }
+
+            else if (command.contains("done")) {
+                markAsDone(tasks, horizLine, command);
+            }
+
+            else {
+                taskCount = computeBasicCommand(tasks, horizLine, command, taskCount);
+            }
+
+            command = in.nextLine();
+        }
+
+        showFarewellMessage(horizLine);
+    }
```

Comment:
The else if and else statement format are not abiding the java coding standards. There are also several unnecessary empty lines in the while loop, consider removing them.